eidpsam/src/test/java/uk/ac/ucl/eidp/auth/jaspic/HttpHeaderAuthModuleTest.java

```diff
@@ -0,0 +1,92 @@
+package uk.ac.ucl.eidp.auth.jaspic;
+
+import java.security.Principal;
+import static org.mockito.Mockito.mock;
```

Comment:
\[Checkstyle\] WARNING: Import statement for 'org\.mockito\.Mockito\.mock' is in the wrong order\. Should be in the 'STATIC' group, expecting not assigned imports on this line\.